examples/language-sdk-instrumentation/golang-push/migrating-from-standard-pprof/main.go

```diff
@@ -74,16 +75,17 @@ func main() {
 			pyroscope.ProfileBlockDuration,
 		},
 	})
+	if err != nil {
+		log.Fatalf("Error starting profiler: %v", err)
+	}
+	defer profiler.Stop()
 
-	var wg sync.WaitGroup
-
-	// Server for pprof
-	go func() {
-		fmt.Println(http.ListenAndServe("localhost:6060", nil))
-	}()
-	wg.Add(1) // pprof - so we won't exit prematurely
```

Comment:
I think it's fair to assume that we don't need it, given that we're migrating to the Pyroscope SDK. OTOH, we don't say anything about it in the README: usually, an existing HTTP server is used and this should not be required.